keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -316,19 +317,19 @@ kind: BucketClass
 metadata:
   name: 
 provisioner: [1]
-protocol: [2]
+supportedProtocols: [2]
 accessMode: {"ro", "wo", "rw"} [3]
 releasePolicy: {"Delete", "Retain"} [4]
-bucketNameActual: [5]
-bucketIdentifier: [6]
+bucketNameActual: [5] // TODO
+bucketIdentifier: [6] // TODO
 secretRef: [7]
   name:
   namespace:
 parameters: string:string [8]
 ```
 
 1. `provisioner`: The name of the driver. If supplied the driver container and sidecar container are expected to be deployed. If omitted the `secretRef` is required for static provisioning.
-1. `protocol`: protocol the associated object store supports (e.g. swift, s3, gcs, etc.). *Only* serves a descriptive purpose and is not verified.
+1. `protocol`: protocols the associated object store supports (e.g. swift, s3, gcs, etc.).  Used in Bucket to BucketClass matching ops.
```

Comment:
_supportedProtocols_

Reply:
That's a good question... the idea is that the protocols be extensible, rather than us picking which are and aren't supported.  So maybe a 4th protocol for yet-to-be-supported fields that is just a string:string map.